operations/reservations.md

```diff
@@ -220,8 +228,7 @@ Returns all reservations specified by any identifier, customer or other filter.
 | `AvailabilityBlockId` | string | optional | Unique identifier of the [Availability block](services.md#availability-block) the reservation is assigned to. |
 | `RateId` | string | required | Identifier of the reservation [Rate](services.md#rate). |
 | `VoucherId` | string | optional | Unique identifier of the [Voucher](services.md#voucher) that has been used to create reservation. |
-| `AdultCount` | number | required | Count of adults the reservation was booked for. |
-| `ChildCount` | number | required | Count of children the reservation was booked for. |
+| `PersonCounts` | Array of [Age category person count](#age-category-person-count) | required | Number of people per age category the reservation was booked for. At least one person must be provided. |
```

Comment:
Regarding the persons/people - I'd use this as we mean a countable group, I have a feeling that people refers to rather uncountable mass of persons.
```suggestion
| `PersonCounts` | array of [Age category person count](#age-category-person-count) | required | Number of persons per age category the reservation was booked for. At least one category with valid count must be provided. |
```

Reply:
Not sure what's wrong with people. https://dictionary.cambridge.org/grammar/british-grammar/person-persons-or-people

Reply:
Ok, eave the people in. Rest of the comment is valid - partner is providing counts, not the real persons.

Reply:
I think you could use 'people' or 'persons' in this case, either would be ok.